client/src/components/presenter/index.jsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import PreGame from './PreGame.jsx';
+import css from '../../../styles/style.css';
```

Comment:
I haven't seen this before so I'm wondering what the reason for importing the CSS here is? We are pulling in the CSS stylesheet in the index_presenter.html and index_player.html so it should be picked up at that point.

Reply:
This commit has a large number of changes to improve the CSS workflow.  css-loader is added to the package, then  Webpack is instructed to recompile on any change to a css file.  The css file is included in the jsx imports so that webpack knows about it.  Anyway, that is my best understanding.  I just learned all this yesterday ;-D

Reply:
Okay. We might be able to get around doing this if we use nodemon during the development to update the server automatically. I'm fine with leaving this for now, but it would be a good idea to explore using nodemon to achieve the same goal.